web/app/js/components/Version.test.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
```

Comment:
I think splitting this into its own file is a good idea. However `Navigation.test.jsx` still has the original tests of `Version`, with the same titles, which would make debugging difficult (for example, `renders update message when versions do not match` now occurs in two files).